content/developer/reference/user_interface/views/include/search.rst

```diff
@@ -0,0 +1,339 @@
+.. _reference/user_interface/views/search:
+
+Search
+======
+
+Search views are a break from previous view types in that they don't display
+*content*: although they apply to a specific model, they are used to filter
+other view's content (generally aggregated views
+e.g. :ref:`reference/user_interface/views/list` or :ref:`reference/user_interface/views/graph`). Beyond that
+difference in use case, they are defined the same way.
+
+The root element of search views is ``<search>``. It takes no attributes.
+
+.. @string is not displayed anywhere, should be removed
+
+Possible children elements of the search view are:
+
+.. rst-class:: o-definition-list
+
+``field``
+    fields define domains or contexts with user-provided values. When search
+    domains are generated, field domains are composed with one another and
+    with filters using **AND**.
+
+    Fields can have the following attributes:
+
+    .. rst-class:: o-definition-list
+
+    ``name``
+        the name of the field to filter on
+    ``string``
+        the field's label
+    ``operator``
+        by default, fields generate domains of the form :samp:`[({name},
+        {operator}, {provided_value})]` where ``name`` is the field's name and
+        ``provided_value`` is the value provided by the user, possibly
+        filtered or transformed (e.g. a user is expected to provide the
+        *label* of a selection field's value, not the value itself).
```

Comment:
I don't understand this part.
I think this needs to be reorganized. I would define the property operator as something like 
"Changes the default operator of a filter. The default operator depends on the field type (e.g. = for float and ilike char). "
Also, do you think of a good use case to use as an example ?

Reply:
I did not improve the doc for this view there...